syntax = "proto3";

package fixbug;
option cc_generic_services = true;
message ResultCode{
    uint32 errncode = 1;
    bytes errmsg = 2;
}

message GetFriendsRequest{
    uint32 id = 1;
}

message GetFriendsResponse{
    ResultCode result = 1;
    repeated bytes friends = 2;
}

service FriendsServiceWanRpc{
    rpc GetFriendLists(GetFriendsRequest) returns(GetFriendsResponse);
}